### ISSUE TITLE
views: support for elasticsearch scroll to retrieve all records

### ISSUE THREAD

**Issue by lnielsen** [inveniosoftware/invenio-records-rest#29]
In order to be able to use the records REST API to retrieve all records we need to add support for scroll over the elasticsearch results.

Related to #28

**Comment by lnielsen** [comment]
@nharraud Is this something you would have time to look into?

**Comment by jma** [comment]
It would be great to integrate this feature as we it is an alternative to the legacy OAI-PMH harvesting.

**Comment by lnielsen** [comment]
Agreed, it would probably need separate rate-limiting. The alternative to this feature is something like the exporter we have in Zenodo, that can produce a full dump of the records on a regular basis. On Zenodo e.g. a full-dump of 400k records is something like 250MB BZip2 compressed JSON.
